### PR TITLE
VACMS-14481: Some tweaks to build-frontend.sh script.

### LINKED ISSUE #14481: Content release broken on D10 preview.
## Description
Reported here: https://dsva.slack.com/archives/C05EJD52H1R/p1689885604146889

Appears to be Tugboat <-> D10 interaction. Will look into it tomorrow.

## Acceptance Criteria
- [x] Content release works on Tugboat.

### PR DESCRIPTION
**Pull request by ndouglas**
Closes #14481.

### CODE REVIEW

File: scripts/build-frontend.sh [hunk 1]
```diff
@@ -53,7 +55,7 @@ rm -rf ${reporoot}/docroot/vendor/va-gov
 composer install --no-scripts &>> ${logfile}
 
 # Get the requested frontend version
-feversion=$(drush va-gov:content-release:get-frontend-version)
```
Comment (ndouglas) [inline scripts/build-frontend.sh:56]: Take last line, which will strip out leading deprecation/warning/error messages, etc.

File: scripts/build-frontend.sh [hunk 2]
```diff
@@ -43,6 +43,8 @@ logfile="${reporoot}/docroot/sites/default/files/build.txt"
 [ -f "${logfile}" ] && rm ${logfile}
 touch ${logfile}
 
+date >> ${logfile}
```
Comment (ndouglas) [inline scripts/build-frontend.sh:46]: Just to confirm that the logfile is from when we think it is.

File: scripts/build-frontend.sh [hunk 3]
```diff
@@ -20,8 +20,8 @@ fi
 cd $reporoot
 
 # Make sure that we're ready to start a build.
-releasestate=$(drush va-gov:content-release:get-state)
-if [ "$releasestate" != "dispatched" ]; then
+releasestate=$(drush va-gov:content-release:get-state | grep -c 'dispatched')
+if [ "$releasestate" -ne 1 ]; then
```
Comment (ndouglas) [inline scripts/build-frontend.sh:24]: Count appearances of the expected string to avoid getting thrown off by error/warning/deprecation messages, etc.

**Review by edmund-dunn — APPROVED** [review]
Good work!

**Pull request merged**